### ISSUE TITLE
Firmware: Develop a driver for the DS2775G+ IC

### ISSUE THREAD

**Issue by mgm8** [spacelab-ufsc/eps2#24]
[https://br.mouser.com/datasheet/2/256/DS2775-DS2778-1513647.pdf](https://br.mouser.com/datasheet/2/256/DS2775-DS2778-1513647.pdf)

**Comment by YanAzer** [comment]
@AugustoVassoler can we close this issue? It may have still compilation errors found when debugging on CSS.

**Comment by AugustoVassoler** [comment]
Yes. The next errors will appear during the test routine run and/or hardware running, and can be referenced in new issues.
